bin/build-all-variants

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env ruby
+
+require "yaml"
+require "optparse"
+require "set"
+
+class Config
+  attr_reader :options
+
+  ##
+  # We need to preserve the order of answers, so we are using nested arrays
+  # rather than a hash. Note that the questions asked in later steps can depend
+  # on the answers to previous steps
+  #
+  VARIANTS = {
+    "basic" => [
+      # [answer, description-of-answer]
+      ["example.com",         "Set production URL"],
```

Comment:
🐘 I feel like this would read better if they were the other way around: `[question: string, answer: string]`.

Reply:
it's not a question and answer - it's the answer and a human descriptionof it so you can keep it straight in your head when you are filling in the answer. It could be a comment I guess but I was looking to future use tbh